### PR TITLE
Add metric for askpass, noop sync != success

### PR DESCRIPTION
**Pull request by thockin**

### CODE REVIEW

File: cmd/git-sync/main.go [hunk 1]
```diff
@@ -129,6 +129,17 @@ var (
 		Name: "git_sync_count_total",
 		Help: "How many git syncs completed, partitioned by success",
 	}, []string{"status"})
+
+	askpassCount = prometheus.NewCounterVec(prometheus.CounterOpts{
+		Name: "git_sync_askpass_calls",
+		Help: "How many git syncs completed, partitioned by success",
```
Comment (stp-ip) [inline cmd/git-sync/main.go:135]: Small nit.
```suggestion
		Help: "How many git askpass calls completed, partitioned by success",
```

File: cmd/git-sync/main.go [hunk 2]
```diff
@@ -371,9 +384,11 @@ func main() {
 			continue
 		} else if changed && webhook != nil {
 			webhook.Send(hash)
```
Comment (stp-ip) [inline cmd/git-sync/main.go:386]: What's the reasoning behind no success increment within the webhook case?

**Review by stp-ip — CHANGES_REQUESTED** [review]

File: cmd/git-sync/main.go (hunk 1 above)
Comment (thockin) [reply cmd/git-sync/main.go:135]: ACK

File: cmd/git-sync/main.go (hunk 2 above)
Comment (thockin) [reply cmd/git-sync/main.go:386]: A bug?  Will fix

**Comment by thockin** [comment]
New push is up

**Review by stp-ip — APPROVED** [review]
lgtm

**Pull request merged**